addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
```

Comment:
Problem with using "ignore" of "stereoscopic playback mode" is it's not a default option, so out-of-the-box, MVC is enabled which means 50% more file/network bandwidth is needed when playing a 3D ISO which is more likely to fail. Yes, I know ISO playback isn't in yet, but it will be one day and will use the same settings as this PR.

Now, I don't watch a lot of 3D Blu-rays, but I assume that many recent Blu-rays support 3D, and I know 3D Blu-Rays are perfectly fine for watching as 2D. It would seem likely that many Blu-rays in a users library would be 3D capable even if the user's TV does not support 3D. It seems having the option to enable/disable 3D support (which has enough overhead that it may be too much for a less than perfect network) is worth having.